### ISSUE TITLE
[Future SuperSlicer versions] Try and fix the bug when using tree/organic supports with adaptive layer height

### ISSUE THREAD

**Issue by VinnyCordeiro** [supermerill/Slic3r#3971]
PrusaSlicer's tree/organic supports are incompatible with the adaptive layer height feature, making the supports the wrong size for the features that needs support. This problem also happen in OrcaSlicer (which has code derived from PrusaSlicer and BambuStudio), but does not occurs on Cura.

Since SuperSlicer will need to catch up in development, this is an issue to look after.

**Comment by basementbuilder17** [comment]
Please add organic supports of any kind for starters.... +1

**Comment by broncosis** [comment]
yes please please add Organic support its the only reason I still go back to Prusa slicer from time to time
